pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -377,11 +377,21 @@ class ScreenAdapter extends EventTarget {
         });
 
         window.addEventListener('resize', (): void => {
-            if (!this.handleResizeEvent) {
-                return;
-            }
+            // if (!this.handleResizeEvent) {
+            //     return;
+            // }
```

Comment:
Why comment these lines?

Reply:
Previously added variables to bypass the issue.